source/AI.cpp

```diff
@@ -1430,6 +1412,8 @@ void AI::MoveIndependent(Ship &ship, Command &command) const
 	const System *origin = ship.GetSystem();
 	if(!ship.GetTargetSystem() && !ship.GetTargetStellar() && !shouldStay)
 	{
+		// TODO: This should problably be changed, because JumpsRemaining
+		// does not return an accurate number.
```

Comment:
Maybe specify what needs to be changed about it? It doesn't consider whormholes?